test/extended/builds/start.go

```diff
@@ -368,7 +368,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds][Slow] starting a build using CL
 				})
 				g.It("Should accept build args that are specified in the Dockerfile", func() {
 					g.By("starting the build with --build-arg flag")
-					br, _ := exutil.StartBuildAndWait(oc, "sample-build-docker-args", "--build-arg=foo=bar")
+					br, _ := exutil.StartBuildAndWait(oc, "sample-build-docker-args", "--build-arg=foofoo=bar")
```

Comment:
so this one is for `sample-build-docker-args` @bparees 

with `sample-build-docker-args-preset` since we are using the default in the BC there are no additional args in the go file to change